docs/user/glossary.rst

```diff
@@ -39,18 +40,15 @@ toggle additional features using the :guilabel:`Tools` menu for each term.
 
 .. image:: /images/glossary-tools.png
 
-Not translatable terms
-++++++++++++++++++++++
+Untranslatable terms
```

Comment:
Oh yea, that is right.

Reply:
It's only one instance, for mouse-over text: https://hosted.weblate.org/translate/weblate/application/en/?checksum=6049b99cdd1c207a

But `Untranslatable` is not used at all... I'm ok if it will be changed in the UI thou.

Reply:
closing as outdated